docs/outlook/index.yml

```
@@ -60,9 +60,9 @@ landingContent:
       - linkListType: reference
         links:
           - text: Ask questions
-            url: https://stackoverflow.com/questions/tagged/office-js
+            url: https://docs.microsoft.com/en-us/answers/topics/office-addins-dev.html
```

You can link to docs.microsoft.com content like this. It allows for the link to be accessible in air-gapped environments.
```suggestion
            url: /answers/topics/office-addins-dev.html
```